build.gradle

```diff
@@ -341,6 +375,51 @@ task dockerRemoveDB(type: com.bmuschko.gradle.docker.tasks.container.DockerRemov
 	targetContainerId { dbContainerName }
 }
 
+// fake-smtp-server tasks
+task dockerPullFakeSmtpServer(type: com.bmuschko.gradle.docker.tasks.image.DockerPullImage) {
+	group = "fake-smtp-server container"
+	repository = "devoto13/fake-smtp-server"
+	description = "Pulls a docker image for the fake-smtp-server from " + repository + "."
+	tag = "0.1.0" // No 'latest' available yet
+}
+
+task dockerCreateFakeSmtpServer(dependsOn: dockerPullFakeSmtpServer, type: com.bmuschko.gradle.docker.tasks.container.DockerCreateContainer) {
+	group = "fake-smtp-server container"
+	imageId = dockerPullFakeSmtpServer.getImageId()
+	containerName = fakeSmtpContainerName
+	description = "Creates a fake-smtp-server container named " + fakeSmtpContainerName + "."
+	exposePorts("tcp", [1025, 1080])
+	portBindings = [run.environment["ANET_SMTP_PORT"] + ":" + 1025, run.environment["ANET_SMTP_HTTP_PORT"] + ":" + 1080]
+}
+
+task dockerStartFakeSmtpServer(type: com.bmuschko.gradle.docker.tasks.container.DockerStartContainer) {
+	group = "fake-smtp-server container"
+	description = "Starts " + fakeSmtpContainerName + " container."
+	targetContainerId { fakeSmtpContainerName }
+}
+
+task dockerStopFakeSmtpServer(type: com.bmuschko.gradle.docker.tasks.container.DockerStopContainer) {
+	group = "fake-smtp-server container"
+	description = "Stops " + fakeSmtpContainerName + " container."
+	targetContainerId { fakeSmtpContainerName }
+}
+
+task dockerRemoveFakeSmtpServer(type: com.bmuschko.gradle.docker.tasks.container.DockerRemoveContainer) {
+	group = "fake-smtp-server container"
+	description = "Removes " + fakeSmtpContainerName + " container."
+	targetContainerId { fakeSmtpContainerName }
+}
+
+// Create the task that runs the integration tests found from the configured source directory and uses the correct classpath.
+task integrationTest(type: Test) {
+	testClassesDirs = sourceSets.integrationTest.output.classesDirs
+	classpath = sourceSets.integrationTest.runtimeClasspath
+
+	// If you want to ensure that integration tests are run every time when you invoke this task, uncomment the following line.
+	outputs.upToDateWhen { false }
```

Comment:
Should this line be commented out? Or should this line and the previous one be deleted altogether?

Reply:
I removed the line